types/@metamask/obs-store.d.ts

```diff
@@ -0,0 +1,18 @@
+import { ObservableStore } from '@metamask/obs-store';
+
+declare module '@metamask/obs-store' {
+  export class ComposedStore<
```

Comment:
Overriding this is still necessary because the types for ComposedStore assume that each store is an object.

Reply:
@Gudahtt Should I try to update `@metamask/obs-store` with these changes?

Reply:
Ah interesting. Yeah I guess so? I see that in the ObservableStore class they made the same mistake in assuming the state was an object, but then fixed it by using generics instead. Applying the same fix to the composable store would make sense.

Reply:
Okay, PRs created here: https://github.com/MetaMask/obs-store/pull/79, https://github.com/MetaMask/obs-store/pull/80